azure-functions/acbs-function/mappings/facility/helpers/get-maximum-liability.js

```diff
@@ -1,4 +1,7 @@
 const getMaximumLiability = (facilitySnapshot) => {
-  return typeof facilitySnapshot.ukefExposure !== 'number' ? Number(facilitySnapshot.ukefExposure.replace(/,/g, '')) : facilitySnapshot.ukefExposure;
+  if (typeof facilitySnapshot.ukefExposure !== 'number') {
+    return Number(facilitySnapshot.ukefExposure.replace(/,/g, ''));
+  }
+  return facilitySnapshot.ukefExposure;
```

Comment:
Looks like there's no test coverage for the ACBS mapping functions - just worried things can break/things don't fail when we change logic 🙃

Reply:
..Probably one for the future or after this PR?

Reply:
I agree, will be adding Cypress test  as part of future PR.

Reply:
Cool, unit tests would be good as well 🚀

Reply:
Yep sounds good mate will do.